packages/itwinui-css/src/button/base.scss

```diff
@@ -40,6 +40,7 @@ $disabledButtonSelectors: '[disabled], :disabled, [aria-disabled="true"], [data-
 
 @mixin iui-button-icon {
   display: inline-flex;
+  flex-shrink: 0;
```

Comment:
We should make another release `3.17.0-dev.1` before this PR merges. We can temporarily add it back for the purpose of that release (otherwise the SideNavigation would look broken).